src/scenedit/scen.global.hpp

```diff
@@ -19,6 +19,9 @@ const int RIGHT_AREA_HEIGHT = 400;
 const int TER_RECT_UL_X = 6;
 const int TER_RECT_UL_Y = 19;
 
+const int UI_LAYER_DEFAULT = 1000;
```

Comment:
Seeing this constants duplicated in two places makes me wonder if they should be in the interface header instead...

Reply:
Layer ids are per application values. And since game and editors are different applications I think this is fine.